tools/imglab/src/main.cpp

```diff
@@ -552,9 +564,19 @@ int tile_dataset(const command_line_parser& parser)
     chdir.revert();
 
     if (ext == "png")
+    {
         save_png(tile_images(images), out_image);
+    }
+    else if (ext == "jpg")
+    {
+        save_jpeg(tile_images(images), out_image, JPEG_QUALITY);
+    }
     else
-        save_jpeg(tile_images(images), out_image);
```

Comment:
jpeg is a better default for this.  It's really small and more importantly dlib will pretty much always end up being built with jpeg support, but not webp support.

Reply:
I thought about it, but to reach this point, the extension must be either ".png", ".jpg", or ".webp", any other extension will cause the program to exit at line 513, (525 after this PR). So, I think it's fine, right? We will always use the extension requested by the user.

Reply:
But maybe I can change the order of the if/else statements if that makes it more clear :)

Reply:
Oh right, yeah duh.  This was fine :D